search2.html

```diff
@@ -43,6 +43,29 @@
 	</select>
 	</div>
 	</div>
+	
+	<div class="col-md-3">
+	<div class="form-group">
+	<label for="topic">Topic</label>
+	<select class="form-control" id="topic">
+		<!-- Manually add a few options -->
+		<option value="Any">Any</option>
+		<option value="GTN FAQ">GTN FAQ</option>
+		<option value="Galaxy FAQ">Galaxy FAQ</option>
```

Comment:
The FAQs aren't really topics, and can already be filtered for in the "Type" dropdown, are they needed here again or should we remove them? What do you think?